packages/design-systems/src/components/composite/Datagrid/types.ts

```diff
@@ -55,6 +56,7 @@ export type UseDataGridProps<TData> = {
   enablePinning?: boolean;
   columnPinning?: ColumnPinningState;
   setColumnPinning?: (updater: Updater<ColumnPinningState>) => void;
+  selectList?: number[];
```

Comment:
この命名だとpaginationと関係があるのか不明なのでMUIに合わせて `pageSizeOptions` でお願いしたいです 🙏 
https://mui.com/x/react-data-grid/pagination/#page-size-options
```suggestion
  pageSizeOptions?: number[];
```